### ISSUE TITLE
Nan value treatment differs: 0 != 1

### ISSUE THREAD

**Issue by schpedee** [catboost/catboost#1271]
This error occurs when using init_model feature. 
Retrain the model
Zero count = 213749, One count = 10236, One Weight = 20.882082844861273
Ratio [0.95430051 0.04569949]
Learning rate set to 0.011431
0:      learn: 0.7464324        test: 0.7248285 best: 0.7248285 (0)     total: 3.81s    remaining: 10h 35m 16s
100:    learn: 0.8991797        test: 0.6986171 best: 0.7462111 (47)    total: 3m 4s    remaining: 5h 1m 57s
Stopped by overfitting detector  (100 iterations wait)

bestTest = 0.7462110928
bestIteration = 47

Traceback (most recent call last):
  File "/opt/fingerprints/supervised/training.py", line 147, in <module>
    verbose=100,
  File "/opt/fingerprints/ml/lib64/python3.6/site-packages/catboost/core.py", line 3959, in fit
    silent, early_stopping_rounds, save_snapshot, snapshot_file, snapshot_interval, init_model)
  File "/opt/fingerprints/ml/lib64/python3.6/site-packages/catboost/core.py", line 1713, in _fit
    train_params["init_model"]
  File "/opt/fingerprints/ml/lib64/python3.6/site-packages/catboost/core.py", line 1229, in _train
    self._object._train(train_pool, test_pool, params, allow_clear_pool, init_model._object if init_model else None)
  File "_catboost.pyx", line 4260, in _catboost._CatBoost._train
  File "_catboost.pyx", line 4306, in _catboost._CatBoost._train
_catboost.CatBoostError: catboost/libs/model/model.cpp:893: Nan value treatment differs: 0 != 1

**Comment by deepesch** [comment]
https://file.io/m5GbYCZn

**Comment by annaveronika** [comment]
Thanks for the report, we'll look into this

**Comment by nofarm3** [comment]
Hi @annaveronika 
I run into the same problem, is there any info I can send that can help you resolve this issue?

**Comment by deepesch** [comment]
Hi @annaveronika, when/how often do you recommend initiating the init_model. Does it require a similar data imbalance? or one can use it on streaming data.

**Comment by vroomzel** [comment]
I got the same error when using `sum_models` (CatBoost 0.22)
```
File "/home/anaconda3/envs/py37/lib/python3.7/site-packages/catboost/core.py", line 4948, in sum_models
    result._sum_models(models, weights, ctr_merge_policy)
  File "/home/anaconda3/envs/py37/lib/python3.7/site-packages/catboost/core.py", line 1354, in _sum_models
    self._object._sum_models(models_inner, weights, ctr_merge_policy)
  File "_catboost.pyx", line 4338, in _catboost._CatBoost._sum_models
  File "_catboost.pyx", line 4348, in _catboost._CatBoost._sum_models
_catboost.CatBoostError: catboost/libs/model/model.cpp:893: Nan value treatment differs: 1 != 0
```

**Comment by kizill** [comment]
We've fixed this in master. Will be released tomorrow in 0.24

**Comment by kizill** [comment]
Shipped in 0.24
